api/proto/teleport/legacy/types/types.proto

```diff
@@ -7812,3 +7812,11 @@ message AccessGraphAWSSync {
   // Integration is the integration name used to generate credentials to interact with AWS APIs.
   string Integration = 4 [(gogoproto.jsontag) = "integration,omitempty"];
 }
+
+// AccessGraphAzureSync is a configuration for Azure Access Graph service poll service.
+message AccessGraphAzureSync {
```

Comment:
Can we postpone this change into PR version?

Reply:
Yes, I can move it to the Azure Discovery PR https://github.com/gravitational/teleport/pull/48843